File: 3830-find-closest-person/find-closest-person.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        if abs(x - z) == abs(y - z):
            return 0
        else:
            if abs(x - z) > abs(y - z):
                return 2
            else:
                return 1

File: 3830-find-closest-person/find-closest-person.cpp
class Solution {
public:
    int findClosest(int x, int y, int z) {
        if (abs<int>(x - z) == abs<int>(y - z)) {
            return 0;
        } else {
            if (abs<int>(x - z) > abs<int>(y - z)) {
                return 2;
            } else {
                return 1;
            }
        }
    }
};